src/hotspot/share/jfr/recorder/checkpoint/types/traceid/jfrTraceIdBits.hpp

```diff
@@ -35,28 +34,28 @@ class JfrTraceIdBits : AllStatic {
   static traceid load(const T* ptr);
 
   template <typename T>
-  static void store(jbyte bits, const T* ptr);
+  static void store(uint8_t bits, const T* ptr);
 
   template <typename T>
-  static void cas(jbyte bits, const T* ptr);
+  static void cas(uint8_t bits, const T* ptr);
 
   template <typename T>
-  static void meta_store(jbyte bits, const T* ptr);
+  static void meta_store(uint8_t bits, const T* ptr);
 
   template <typename T>
-  static void mask_store(jbyte mask, const T* ptr);
+  static void mask_store(uint8_t mask, const T* ptr);
 
   template <typename T>
-  static void meta_mask_store(jbyte mask, const T* ptr);
+  static void meta_mask_store(uint8_t mask, const T* ptr);
 
   template <typename T>
-  static void clear(jbyte bits, const T* ptr);
+  static void clear(uint8_t bits, const T* ptr);
 
   template <typename T>
-  static void clear_cas(jbyte bits, const T* ptr);
+  static void clear_cas(uint8_t bits, const T* ptr);
 
   template <typename T>
-  static void meta_clear(jbyte bits, const T* ptr);
+  static void meta_clear(uint8_t bits, const T* ptr);
```

Comment:
jbyte -> uint8_t is definitely preferable.

Reply:
Yes.